saleor/order/notifications.py

```diff
@@ -16,16 +17,82 @@
     from ..account.models import User  # noqa: F401
 
 
+def get_image_payload(instance: ProductImage):
+    image_file = instance.image if instance else None
+    return {
+        size: get_thumbnail(image_file, size, "thumbnail")
+        for size in AVAILABLE_PRODUCT_SIZES
+    }
+
+
+def get_default_images_payload(images: List[ProductImage]):
+    first_image_payload = None
+    first_image = images[0] if images else None
+    if first_image:
+        first_image_payload = ({"original": get_image_payload(first_image)},)
+    images_payload = None
+    if images:
+        images_payload = [{"original": get_image_payload(image) for image in images}]
+    return {"first_image": first_image_payload, "images": images_payload}
```

Comment:
Do we need to return the first image separately? `images` contains it as well.

Reply:
We were thinking about this, but the first_image gives you an easier interface to build email content

Reply:
👍